pkg/provision/buildroot.go

```diff
@@ -170,7 +170,7 @@ func setRemoteAuthOptions(p provision.Provisioner) auth.Options {
 	dockerDir := p.GetDockerOptionsDir()
 	authOptions := p.GetAuthOptions()
 
-	// due to windows clients, we cannot use filepath.Join as the paths
+	// due to windows clients, we cannot use path.Join as the paths
```

Comment:
I think you want this comment to stay the same

Reply:
done